libeos-updater-util/flatpak.c

```diff
@@ -121,6 +121,18 @@ euu_flatpak_remote_ref_action_unref (EuuFlatpakRemoteRefAction *action)
   g_slice_free (EuuFlatpakRemoteRefAction, action);
 }
 
+char *
+euu_flatpak_remote_ref_action_get_collection_id (EuuFlatpakRemoteRefAction *action)
+{
+    return g_strdup (action->ref->collection_id);
+}
+
+char *
+euu_flatpak_remote_ref_action_get_ref (EuuFlatpakRemoteRefAction *action)
```

Comment:
I’d call this `euu_flatpak_remote_ref_action_format_ref()`, since it’s returning the string formatted version of the ref, not the `FlatpakRef*` object.

Same comments from above about `gchar` and documentation apply here. Please make sure to mention that the formatted ref is generated by `flatpak_ref_format_ref()` and hence can be parsed by `flatpak_ref_parse()`.